pom.xml

```diff
@@ -90,11 +90,12 @@
     <logback.version>1.2.3</logback.version>
     <maven.project.version>2.2.1</maven.project.version>
     <mockito.version>4.11.0</mockito.version>
-    <sonar.version>9.7.1.62043</sonar.version>
-    <sonar.orchestrator.version>3.40.0.183</sonar.orchestrator.version>
-    <sonar.api.version>9.11.0.290</sonar.api.version>
-    <sonar-analyzer-commons.version>2.5.0.1358</sonar-analyzer-commons.version>
-    <sonarlint-core.version>8.14.0.63103</sonarlint-core.version>
+    <sonar.version>9.9.1.69595</sonar.version>
+    <sonar.orchestrator.version>4.0.0.404</sonar.orchestrator.version>
+    <sonar.api.version>10.1.0.809</sonar.api.version>
+    <sonar.plugin.api.version>10.0.0.695</sonar.plugin.api.version>
```

Comment:
why do we need two properties? I replaced usages of `sonar.plugin.api.version` with `sonar.api.version` and everything is still green